README.md

```diff
@@ -51,8 +51,12 @@ Report issues at the [GitHub issue tracker](https://github.com/cision/rover-ui/i
 
 ## Champions
 
+Champions are admins, but they're also people who've been involved for a while
+and have a lot of investment in the project.
+
 The main RoverUI champions currently are:
 
+- [@dkordik](https://github.com/dkordik)
```

Comment:
I'm not 100% comfortable gatekeeping this, anyone with admin rights can do most of the approval stuff.
But I think Dan's demonstrated that he intuitively "gets" the goal of simple, dumb, presentational components, so I added him.

Plenty of other folks have done good work, so let me know if I you want me to add you too!

Reply:
After looking at the contributors section, I see quite a few people have made lots of commits, but the "who's most engaged" factor changes dramatically over time.

I think I'll remove this section, unless we come up with a more concrete example of what being a "champion" means.

Reply:
removing it sounds good to me 👍